docs/howto/overriding-templates.txt

```diff
@@ -0,0 +1,88 @@
+====================
+Overriding Templates
+====================
+
+Overriding from the project's templates directory
+=================================================
+
+To override templates from a package in ``django.contrib`` or from a third-party
+package, the preferred method is to create replacement templates in your
+project's templates directory.
+
+Let's say you're trying to override the templates for a third-party package
+called ``blog``, which provides the templates ``blog/post.html`` and
+``blog/list.html``. The relevant settings for your project would look like::
+
+    INSTALLED_APPS = [
+        ...,
+        blog,
+        ...,
+    ]
+
+    TEMPLATES = [
+        {
+            'BACKEND': 'django.template.backends.django.DjangoTemplates',
+            'DIRS': [os.path.join(BASE_DIR, 'templates')],
+            'APP_DIRS': True,
+            ...
+        },
+    ]
+
+The :setting:`TEMPLATES` section here should already exist in your project
```

Comment:
section -> setting
should already exist -> will already exist if you created your project using the default project template.